lib/pengine/pe_output.c

```diff
@@ -1901,7 +1901,7 @@ node_xml(pcmk__output_t *out, va_list args) {
                 node_type = PCMK_VALUE_REMOTE;
                 break;
             case node_ping:
-                node_type = "ping";
+                node_type = PCMK__VALUE_PING;
```

Comment:
We could make this public since it may appear in API output and is in the API schema as an enumerated value. I'm on the fence. It **shouldn't** appear, given that it's deprecated. If it does appear, I don't know that anyone would need to fetch "ping" nodes specifically. Seems like they could just do "if not member and not remote" since there's also an "unknown" value.